nss-cohort-36/bangazon-client-virtual-magicians-1
# Jg product 1

# Description
This is for creating a Product Form that will be needed for users to post their products to sell.

Closes #[Enter issue #]
## Type of change

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-product-1`
4. go to our homepage and through the nav select the Sell link to be redirected to the form. You should be able to see 5 fields to enter in. The submit button does nothing yet.  
# Checklist:
